### PR TITLE
add governance docs

### PR DESCRIPTION
**Pull request by robert-milan**
This PR will add the initial Governance documents for Metrictank, as well as list the current members.

### CODE REVIEW

File: GOVERNANCE.md
Comment (Dieterbe) [inline GOVERNANCE.md]: what's this header for?

File: GOVERNANCE.md
Comment (Dieterbe) [inline GOVERNANCE.md]: i note that this means we'll need to leave most editorial PR's open for at least 21 days

File: GOVERNANCE.md
Comment (Dieterbe) [inline GOVERNANCE.md]: I think every occurence of `[team]` should be `[teamlist]`

File: GOVERNANCE.md
Comment (Dieterbe) [inline GOVERNANCE.md]: for new members, they will simply see a login prompt here - try it in incognito mode -
```suggestion
[slack]: https://grafana.slack.com ([signup form](http://slack.raintank.io/))
```

File: GOVERNANCE.md
Comment (Dieterbe) [inline GOVERNANCE.md]: When is any PR a "technical issue on which a decision should be made" ?
If all/most of them are, I just realized that this may significantly slow down our process.
This would require either explicit signoff from all the team members or expiration of the 21day timeout, until we can merge PR's.

That doesn't seem practical.

File: GOVERNANCE.md
Comment (Dieterbe) [inline GOVERNANCE.md]: does this allow them to see past threads or only emails since after they joined?

**Review by Dieterbe — CHANGES_REQUESTED** [review]
mainly i want to clear up the decision making process around merging PR's

File: GOVERNANCE.md
Comment (robert-milan) [reply GOVERNANCE.md]: Unless consensus is reached before that time period

File: GOVERNANCE.md
Comment (robert-milan) [reply GOVERNANCE.md]: I'll fix it up and push a commit

File: GOVERNANCE.md
Comment (robert-milan) [reply GOVERNANCE.md]: I copied it over from the source, it has to do with css I believe, which we don't have, so I'll remove it.

File: GOVERNANCE.md
Comment (robert-milan) [reply GOVERNANCE.md]: I think that would make it cluttered with links when there are already plenty which link to the team members document.

File: GOVERNANCE.md
Comment (robert-milan) [reply GOVERNANCE.md]: I believe they have full access to the group at that point, so I think the answer is yes.

File: GOVERNANCE.md
Comment (robert-milan) [reply GOVERNANCE.md]: If we only have 5 team members then this would require explicit signoff from 3 for majority or 4 for supermajority, at which point the last person's vote doesn't matter.

Maybe we should change it back to 14 days then, or try to better define categories of changes. It's a bit dangerous to get into specifics though because it is a very grey area.

That being said, I don't think voting will affect many PRs. Voting and consensus are around the idea to change something, not necessarily the actual change. Once a decision has been made / voted upon and a PR is opened do we want to vote on that PR then? If not, then as long as the issue or thread in the dev mailing list already passed with either lazy consensus or a formal vote, there is no need to use this process.

File: GOVERNANCE.md
Comment (Dieterbe) [reply GOVERNANCE.md]: Huh?
1) My suggestion does not change the amount of links. You already have links.
2) `[team]` is a broken link. There is no entry "team" in the list below (only "teamlist") and so your links don't work

Comment (robert-milan) [reply GOVERNANCE.md]: Ahhh thanks @Dieterbe

File: GOVERNANCE.md
Comment (Dieterbe) [reply GOVERNANCE.md]: can you find out how this is dealth with in other projects?

File: GOVERNANCE.md
Comment (Dieterbe) [reply GOVERNANCE.md]: Yes but that's kindof ridiculous, any time someone files a PR for an editorial tweak, it's silly that either we need input from many team members, or wait 21 days

File: GOVERNANCE.md
Comment (Dieterbe) [inline GOVERNANCE.md]:
```suggestion
A pull request which proposes changes aligned with an approved technical decision does not require consensus. Merely the approval of one team member who is not the author of the PR. If a team member explicity objects to the change (whether before or after the PR) a vote may be called (to block or revert the PR, respectively).
```

File: GOVERNANCE.md
Comment (Dieterbe) [reply GOVERNANCE.md]: The change to 5 days looks good.

File: GOVERNANCE.md
Comment (Dieterbe) [reply GOVERNANCE.md]: I like the new wording around PR's. thanks

**Review by Dieterbe — APPROVED** [review]

**Pull request merged**
